docs/source/glossary.md

```diff
@@ -3,20 +3,18 @@
 A glossary of common terms used throughout the PyMC documentation and examples.
 
 :::::{glossary}
+
```

Comment:
There should be no space between the directive start and it's arguments (i.e. the `:sorted:` thing)